pkg/controllers/management/clusterstats/statsaggregator.go

```diff
@@ -63,6 +65,10 @@ func (s *StatsAggregator) aggregate(cluster *v3.Cluster, clusterName string) err
 	var machines []*v3.Node
 	// only include worker nodes
 	for _, m := range allMachines {
+		// if none are set, then nodes syncer has not completed
+		if !m.Spec.Worker && !m.Spec.ControlPlane && !m.Spec.Etcd {
+			return errors.Errorf("Not able to determine node role for %s.", m.Status.NodeName)
```

Comment:
Include something about it not being ready yet since it is probably not in a stuck state.

Reply:
Also, I thought the linter didn't like errors that start with a capital but maybe not.

Reply:
removed capital and expanded error message to include indication of retry